Makefile

```diff
@@ -12,7 +12,7 @@ REGISTRY ?= quay.io
 REGISTRY_ACCOUNT ?= kubev2v
 REGISTRY_TAG ?= devel
 
-VERSION ?= 2.4.0
+VERSION ?= 99.0.0
```

Comment:
incorrect conflict resolution (also below)?

Reply:
yeap,  didnt notice the changes in Makefile, fixed